packages/affiliates/payouts/deployers/2020-11-09_2020-11-16.json

```diff
@@ -0,0 +1,32 @@
+{
+  "config": {
+    "empWhitelist": [
+      "0xaBBee9fC7a882499162323EEB7BF6614193312e3",
+      "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56",
+      "0x306B19502c833C1522Fbc36C9dd7531Eda35862B",
+      "0x3a93E863cb3adc5910E6cea4d51f132E8666654F"
+    ],
+    "startTime": 1604962800000,
+    "endTime": 1605567600000,
+    "totalRewards": "50000",
+    "network": 1
+  },
+  "startBlock": { "timestamp": 1604962804000, "number": 11226178 },
```

Comment:
do we want to convert these from js timestamps to unix timestamps? (1000x smaller).

Reply:
i think if we made the change for this part of the code it would be easy, but it would be inconsistent with the config timestamp format above it.

if we are to do this, id like to keep all timestamps consistent S vs MS. would not be a hard change necessarily, but tedious. would require updating big query queries and any timestamp conversions with moment as well as potentially example scripts and config files to reflect seconds instead of ms. 

if you are OK with the inconsistent output then i can make the change to just that. WDYT?